server/spec/requests/login_spec.rb

```diff
@@ -0,0 +1,21 @@
+# spec/requests/categories_request_spec.rb
+RSpec.describe 'Login', type: :request do
+    # initialize test data
+    let!(:test_user) { FactoryBot.create(:user) }
+    #  POST /category
+    describe 'POST /login' do
+        let(:login_payload) { { user: { username: test_user.username, password: test_user.password } } }
+      context 'Valid Login Payload' do
+        before { post '/login', params: login_payload }
+        it 'User can login successfuly' do
+          json = JSON.parse(response.body);
+          expect(json['user']).not_to eq(nil)
+          expect(json['logged_in']).to eq(true)
+        end
+        it 'returns status code 200' do
+          expect(response).to have_http_status(200)
+        end
+      end
+    end
+
+   end
```

Comment:
I'm not too familiar with this method but it does look good. Maybe add a test for invalid loads for this path and valid/invalid loads for other paths (is_logged_in, logout)?

Also, once you revert the 'simplecov' stuff in spec_helper.rb, coverage reports will be created under coverage directory. You can check that to make sure all paths are visited.

Reply:
> Maybe add a test for invalid loads for this path and valid/invalid loads for other paths (is_logged_in, logout)?

Please add tests for is_logged_in and logout